cypress/support/commands.js

```diff
@@ -118,6 +117,16 @@ Cypress.Commands.add("datasetMeetsTestCriteria", (p_pageName, p_datasetConfig, p
     return true;
 });
 
+// Calls mutation in the Nuxt store
```

Comment:
Oh that's neat! Is this directly commiting to the vuex store? Or is the "Nuxt store" a separate thing?

Reply:
Nope. Nuxt store is meant to be synonymous with Vuex store here. I could just change the function names.

But more importantly - yes! We are committing directly to the store here. Before I just had the function to `dispatch` a store action. In this case, the `commit` call became needed since we refactored the store to more often call mutations directly.

Reply:
Instances of 'NuxtStore' have been changed to 'VuexStore' - including in our testing contribution documentation (https://github.com/neurobagel/annotation_tool/pull/330/commits/f839304315dae34546b3e94c42ec262270d5776c)